package.json

```diff
@@ -28,7 +28,7 @@
   "dependencies": {
     "bootstrap": "^3.3.6",
     "metal-soy": "^2.0.0",
-    "metal-input": "^1.0.3"
+    "metal-input": "^1.0.5"
```

Comment:
There's no need for commits with changes like that. When you specify in package.json that `metal-input`'s version is `^1.0.3` it will automatically install the most recent version of `metal-input` when you reinstall `node_modules`, as long as that version hasn't changed the major version (first version number). So in this case even if you leave this as `^1.0.3` and reinstall you'll see that `metal-input` will be installed in its latest version.

Can you remove the commits that are just doing this kind of update from this pull request? It will allow the commit history to be simpler :)